### PR TITLE
SLM-231 skeleton code

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**

### CODE REVIEW

File: server/app.ts [hunk 1]
```diff
@@ -77,10 +82,16 @@ export default function createApp(
   // no authentication
   app.get('/privacy-policy', (req, res) => res.render('pages/privacy-policy/privacy-policy'))
   app.use('/start', setupLegalSenderStartPage())
-  app.use('/link', requestLinkAuthorised())
-  app.use('/link', setUpLink(app, magicLinkService, appInsightsClient))
   app.use('/contact-helpdesk', setupContactHelpdesk(zendeskService))
 
+  if (config.featureFlags.lsjOneTimeCodeAuthEnabled) {
```
Comment (nathanrussell-moj-digital) [inline server/app.ts:87]: The feature flag will only enable 1 set of routes, which means this is the only bit of code that needs to know about the feature flag

File: server/config.ts [hunk 2]
```diff
@@ -136,9 +137,12 @@ export default {
     legalSenderJourney: get('LEGAL_SENDER_PHASE_BANNER_LINK', 'https://eu.surveymonkey.com/r/SendLegalMail'),
     mailRoomJourney: get('MAIL_ROOM_PHASE_BANNER_LINK', 'https://eu.surveymonkey.com/r/CheckRule39Mail'),
   },
-  fileUploadsEnabled: toBoolean(get('FILE_UPLOADS_ENABLED', false)),
   smoketest: {
     msjSecret: get('APP_SMOKETEST_MSJSECRET', null),
     lsjSecret: get('APP_SMOKETEST_LSJSECRET', null),
   },
+  featureFlags: {
+    fileUploadsEnabled: toBoolean(get('FILE_UPLOADS_ENABLED', false)),
```
Comment (nathanrussell-moj-digital) [inline server/config.ts:145]: I created a `featureFlags` object to contain all our feature flags (keeps them all together), so moved the existing  `fileUploadsEnabled` into it

Comment (mikehalmamoj) [reply server/config.ts:145]: Interestingly I saw that the backlog ticket about file uploads has been rejected - we should think about removing this feature flag... but nothing to do with this story 😆

Comment (nathanrussell-moj-digital) [reply server/config.ts:145]: Yes, definitely - I'll add a ticket for it

**Review by mikehalmamoj — APPROVED** [review]

File: server/config.ts (hunk 2 above)
Comment (nathanrussell-moj-digital) [reply server/config.ts:145]: SLM-232 created

**Pull request merged**